.github/workflows/ci.yml

```diff
@@ -55,8 +55,11 @@ jobs:
           - prod
         test:
           - .github/smoke-test.sh
+          - make lint
           - make test
         exclude:
+          - target: prod
```

Comment:
Separately we can evaluate how excluding `prod` vs. including `dev` looks like when more targets are in place

Reply:
Though at the moment they are not symmetrical, as `exclude` look like the right languages for a matrix whereas `include` modifies rather than adding